.devcontainer/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.6.3
+FROM ruby:2.7.1
```

Comment:
Local Dev was not running with the same Ruby version as CI, so I fixed that here